setup.py

```diff
@@ -32,7 +32,7 @@
 else:
     __version__ = "0.0.dev0"
 
-readme_file = path.join(__dir__, 'README.md')
+readme_file = path.join(__dir__, 'README.rst')
```

Comment:
On line 54, you need to update the `long_description_content_type` value.

Reply:
What should I change it to?

Reply:
https://packaging.python.org/specifications/core-metadata/#description-content-type-optional

Reply:
Oh sorry I was looking at the wrong thing, thought you wanted me to change `description` on line 54.

Reply:
GitHub doesn't let you put comments on unmodified lines it seems :-(